cmd/kflex/ctx/ctx.go

```diff
@@ -106,6 +106,17 @@ func (c *CPCtx) loadAndMergeFromServer(kconfig *api.Config) error {
 		return fmt.Errorf("control plane not found on server: %s", err)
 	}
 
+	// for control plane of type host just create context alias and return
+	if cp.Spec.Type == tenancyv1alpha1.ControlPlaneTypeHost {
+		err = kubeconfig.CopyHostContextAndSetItToDefault(kconfig, c.Name)
```

Comment:
Why make a copy? Why not use the original?

Reply:
The copy of the context or "alias" context is part of the current UX, and it is done mainly for a consistent experience - the idea is that there is a context for each control plane with the same name of the control plane, independently of the type of control plane.

Reply:
@MikeSpreitzer, let me see if I understand the proposal - if the control plane is of type host the idea not to create an alias context (as a copy of the hosting cluster context) with the name of the control plane, but just use the hosting cluster context. That is, `kflex ctx <cp-of-type-host>` is equivalent to `kflex ctx`.  That may make possibly simpler understanding what is going on when using a cp of type host. Is that correct ?

Reply:
yes.
Along with making `kflex ctx` set the hosting context name extension IF it is not already set and the current context IS the hosting context.